requirements/runtime.txt

```diff
@@ -48,10 +48,6 @@ jaraco-context==5.3.0
     # via keyring
 jaraco-functools==4.0.1
     # via keyring
-jeepney==0.8.0
```

Comment:
This one is still in the tree: https://inspector.pypi.io/project/keyring/25.6.0/packages/d3/32/da7f44bcb1105d3e88a0b74ebdca50c59121d2ddf71c9e34ba47df7f3a56/keyring-25.6.0-py3-none-any.whl/keyring-25.6.0.dist-info/METADATA#line.18

Reply:
huh, weird. I just ran the pip-compile command from the header 🤔

Reply:
ah, `sys_platform == "linux"`, of course 🤦🏻